README.md

```diff
@@ -1,3 +1,46 @@
-# massa-as-sdk
+# Massa-as-sdk
 
-Massa Assemblyscript SDK
+_Massa Assemblyscript SDK_
+
+Collection of tools, objects and functions, that are specific in usage only for Massa smart contracts in AssemblyScript 
+
+SDK enables us to import object classes (such as address, storage objects) and use them without having to write all objects from scratch each time.
+
+In order to call ABI functions, we import those functions in the smart-contract project, from the SDK dependency.
+SDK is a set of tools, objects and functions, that are specific in usage only for Massa blockchain.
+
+Contract standards (ERC-20, etc) have been removed from it.
```

Comment:
I'd remove this. We don't need to say in the documentation that we removed some parts. We will mention such things in the releases.